noobaa/noobaa-core
# Add cloud identity to pool cloud info

Signed-off-by: Kfir Payne <kfirpayne@gmail.com>

related https://github.com/noobaa/noobaa-core/issues/5750

### Explain the changes
1.  Add `identity` field in the `info.cloud_info` returned object from `read_system()`.
2. Add the same field in the appropriate json schema.

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1. Verify the Resources tab in the UI shows the resources properly.
2. Verify a specific cloud Resource tab in the UI shows the resource info properly.
